test/08-tasks.test.ts

```diff
@@ -349,7 +349,7 @@ describe('08 task tests', function () {
 			const cron = '0 0 10 1 1 *';
 
 			// Test both succeeded and failure cases
-			['succeeded', 'failed'].forEach(async (status) => {
+			for (const status of ['succeeded', 'failed']) {
```

Comment:
This is the linting fix, the rest is just making the test succeed now that it is actually run properly